bin/create_user

```diff
@@ -1,16 +1,19 @@
 #!/usr/bin/env ruby
 
-require 'digest'
 require 'mongo'
+require 'cherry_silt'
 
-mongo_client = Mongo::Client.new([ '127.0.0.1:27017' ], database: 'mud')
+client = Mongo::Client.new(['127.0.0.1:27017'], database: 'mud')
+
+client[:player].drop
```

Comment:
I'm just using bin/create_user as a way to put stuff in the database for testing. I guess I could make it a full fledged utility that ships with the gem